### DISCUSSION TITLE
can't figure out default malloc in zmalloc.c

### DISCUSSION THREAD

**Discussion by osevan** [dragonflydb/dragonfly#94] [Q&A] [answered]
What is default allocator of dfdb? 

And in this file I see tcmalloc, mimalloc and jemalloc, how I can use different allocator for my home lab tests? 

Need I pass command to blaze? 

Thanks and

Best regards

**Comment by romange** [comment, marked as answer]
we use mimalloc and it's currently unconfigurable, i.e. dragonfly is committed to mimalloc as part of its design. 
everything else is just code remnants from Redis code.
